packages/firebase_messaging/README.md

```diff
@@ -61,77 +61,97 @@ for more.
 
 By default background messaging is not enabled. To handle messages in the background:
 
+1. Add the `com.google.firebase:firebase-messaging` dependency in your app-level `build.gradle` file that is typically located at `<app-name>/android/app/build.gradle`.
+
+   ```gradle
+   dependencies {
+     // ...
+   
+     implementation 'com.google.firebase:firebase-messaging:20.1.0'
```

Comment:
Could you update this to suggest the use of the latest version of the dependency?

`implementation 'com.google.firebase:firebase-messaging:<latest_version>'`

Reply:
Sure, I also added a note on how to obtain the latest version number.